src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -172,10 +173,97 @@ public async Task GetAsync_SetCookieContainerMultipleCookies_CookiesSent()
             };
         }
 
+        [OuterLoop("Uses delays")]
```

Comment:
Does it actually need to be outerloop?  How long does it take to run?  From a quick skim of the test, I don't see any explicit time delays being introduced.

Reply:
It uses a remote server.